src/main/scala/com/quartethealth/spark/fixedwidth/FixedwidthRelation.scala

```diff
@@ -0,0 +1,56 @@
+package com.quartethealth.spark.fixedwidth
+
+import com.databricks.spark.csv.CsvRelation
+import com.databricks.spark.csv.readers.{BulkReader, LineReader}
+import com.quartethealth.spark.fixedwidth.readers.{BulkFixedwidthReader, LineFixedwidthReader}
+import org.apache.spark.rdd.RDD
+import org.apache.spark.sql.SQLContext
+import org.apache.spark.sql.types.StructType
+
+class FixedwidthRelation protected[spark] (
+    baseRDD: () => RDD[String],
+    fixedWidths: Array[Int],
+    location: Option[String],
+    useHeader: Boolean,
+    parseMode: String,
+    comment: Character,
+    ignoreLeadingWhiteSpace: Boolean,
+    ignoreTrailingWhiteSpace: Boolean,
+    treatEmptyValuesAsNulls: Boolean,
+    userSchema: StructType,
+    inferSchema: Boolean,
+    codec: String = null,
+    nullValue: String = "")(@transient override val sqlContext: SQLContext)
+  extends CsvRelation(
```

Comment:
How does it work that this is extending the csv parser? I see code like this https://github.com/quartethealth/spark-csv/blob/master/src/main/scala/com/databricks/spark/csv/CsvRelation.scala#L80-L86 and wonder if extension is the right choice here. At the very least it seems sort of unintuitive to me that a fixed-width parser is a case of a csv parser.